workflow/controller/pod/significant.go

```diff
@@ -1,11 +1,14 @@
 package pod
 
 import (
+	"os"
+
 	apiv1 "k8s.io/api/core/v1"
 )
 
 func SignificantPodChange(from *apiv1.Pod, to *apiv1.Pod) bool {
-	return from.Spec.NodeName != to.Spec.NodeName ||
+	return os.Getenv("ALL_POD_CHANGES_SIGNIFICANT") == "true" ||
```

Comment:
Flag is off by default? Would we want it on by default?

Reply:
this is actually covered by a test, but good to ask